zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -38,9 +40,33 @@ def parse_cbd_path(path):
+
+    return r.clusters[0].UUId
+
+
+def get_version():
+    return shell.call("%s --version | awk '{print $2}'" % ZBS_BIN_PATH).strip()
+
 
-def deploy_client(ip, password):
-    return shell.call("%s client deploy --host %s -p %s --silent" % (ZBSADM_BIN_PATH, ip, linux.shellquote(password)))
+def deploy_client(ip, port, username, password):
+    return shell.call("%s client deploy --host %s --port %s -u %s -p %s --silent" % (ZBSADM_BIN_PATH, ip, port, username, linux.shellquote(password)))
```

Ruff (0.12.2)

`shell` may be undefined, or defined from star imports

(F405)